src/plugins/cordova-plugin-battery-status/sim-host-panels.html

```diff
@@ -1,13 +1,13 @@
 <!-- Copyright (c) Microsoft Corporation. All rights reserved. -->
 
-<cordova-panel id="battery-status" caption="Battery Status" data-loc-id="ac1dbd34">
+<cordova-panel id="battery-status" caption="Battery Status" spoken-text="Battery Status Panel" data-loc-id="ac1dbd34">
     <cordova-group>
         <cordova-panel-row>
-            <cordova-label for="battery-level" label="Battery level" data-loc-id="e0d4e236"></cordova-label>
+            <cordova-label spoken for="battery-level" label="Battery level" data-loc-id="e0d4e236"></cordova-label>
```

Comment:
Is the `spoken` attribute needed here because it isn't read with the input element?